judoscale-rails/lib/judoscale/rails/railtie.rb

```diff
@@ -41,12 +41,6 @@ def judoscale_config
         app.middleware.insert_before Rack::Runtime, RequestMiddleware
       end
 
-      initializer "judoscale.utilization_middleware", after: :load_config_initializers do |app|
```

Comment:
@carlosantoniodasilva I can't think of any reason to make utilization tracking optional, so I've combined the middlewares as you suggested. We no longer need to worry about `:load_config_initializers`, right?